src/view/com/modals/LinkWarning.tsx

```diff
@@ -86,7 +86,7 @@ export function Component({text, href}: {text: string; href: string}) {
             }}
             accessibilityLabel={_(msg`Cancel`)}
             accessibilityHint=""
```

Comment:
And here, what about "Don't open the linked website"?
```suggestion
            accessibilityHint={_(msg`Don't open the linked website`)}
```